ROADMAP.md

```diff
@@ -1,7 +1,18 @@
 # Kubeflow Roadmap
 
-## Kubeflow 1.7 Release - planning now in process 
-* Please join the Kubeflow Community Meetings on Tuesdays for updates and for opportunities to contribute.  
+## Kubeflow 1.7 Release, Planned: March 2023 
+The Kubeflow Community plans to deliver its v1.7 release in March 2023, per this [timeline](https://github.com/kubeflow/community/pull/573).   The high level deliveries are being tracked in this [Project Board](https://github.com/orgs/kubeflow/projects/50/views/1).   The v1.7 release process will be managed by the v1.7 [Release Team](https://github.com/kubeflow/internal-acls/pull/576) using the best practices in the [Release Handbook](https://github.com/kubeflow/community/blob/master/releases/handbook.md)
+
+Notable feature candidates in the [Project Board](https://github.com/orgs/kubeflow/projects/50/views/1) are:
+* Support for Kubernetes 1.25
+* Improved user isolation especially for the Kubeflow pipelines user interface, database, and artifacts
+* Update Kubeflow Noteboks naming from Notebooks to Workbenches
+* Delivery of KFP V2 beta with its new front-end, backend and SDK 
+* New, time saving workflows to simplify data exchange from Katib to Kubeflow Pipelines
+* Distributed Training Operator support for PaddlePaddle
+* Proposals and potential alpha integrations with popular open source projects: MLflow via Concurrent, BentoML, Ray
```

Comment:
I suggest to avoid mentioning specific open source projects' names, so we can be more open in terms of integrations with open source projects in general.

It is probably better to define a set of policies and guidelines first, before introducing those integrations. Because it potentially increase the technical workload for maintaining such integration, if we don't standardize the integration guide.

Reply:
Hi @zijianjoy - Have you seen this update for the contrib components ?  https://github.com/kubeflow/manifests/pull/2286

Reply:
and this one ?  https://github.com/kubeflow/manifests/issues/2311.   I think these are moving in the direction that you identify.

Reply:
It depends on the kind of integration, some projects can fit in '/contrib' folder, while others can serve outside of Kubeflow GitHub organization. 

If we accept all sorts of integrations to '/contrib', the manifests itself will become too hard to maintain (there is no way to test all scenarios because users can choose any combination of addons to install, and problems may occur with high cost to reproduce and fix)

Regarding the roadmap, I cheer for the effort to seek integration with other open source projects, but I would avoid mentioning specific projects because we haven't made decision to integrate or how to integrate with those projects.

Reply:
@zijianjoy on the integration with other open source projects, I would suggest that we haven't required a community wide decision on the integration for previous projects - fairing, kale, seldon, feast, elyra, pachyderm, etc.   From my memory (which may be incomplete), folks just started the integration and did their best with the support they could get.   With the current batch of integrations, we have tried to set expectations and some process: tracking issues have been opened and commented on, and the contributors are joining community / working group calls and design docs reviewed.   I propose that the lack of adoption by some of the previous integrations is (partly) due to the lack of exposure of the work to the community.   I propose the wording on the roadmap is reflective of the state of the integrations...Proposals and potential alpha integrations.    I guess the question then becomes, what happens if the proposals or integrations fail to deliver and how do we message that i.e. potentially via depreciation notices, which is process documentation that is being discussed.

Reply:
I removed the reference to Ray, Bento and Concurrent.